.eslintrc.json

```diff
@@ -1,3 +1,9 @@
 {
-  "extends": "eslint-config-edx"
+  "extends": "eslint-config-edx",
+  "globals": {
```

Comment:
Here's the full list from the old .jshintrc. I wonder if we should proactively bring a few of these in too.

```
        // JavaScript global libraries
        "Backbone",
        "jQuery",
        "$",
        "_",

        // RequireJS globals
        "define",
        "require",
        "RequireJS",

        // Jasmine globals
        "jasmine",
        "describe", "xdescribe",
        "it", "xit",
        "spyOn",
        "beforeEach",
        "afterEach",
        "expect",
        "waitsFor",
        "runs",

        // jQuery-Jasmine globals
        "loadFixtures",
        "appendLoadFixtures",
        "readFixtures",
        "setFixtures",
        "appendSetFixtures",
        "spyOnEvent",

        // Django i18n catalog globals
        "interpolate",
        "gettext",
        "ngettext",

        // Miscellaneous globals
        "JSON",

        // edX globals
        "edx",
        "XBlock"
```

``edx`` and ``XBlock`` seem to be good candidates, as well as the other Jasmine related functions (``describe``, ``it`` etc).